Source/FullScreenPokemon.ts

```diff
@@ -8900,7 +8907,11 @@ module FullScreenPokemon {
         }
 
         /**
+         * Scrolls the game window vertically
+         * until the Map is centered based on
+         * the Area.
```

Comment:
You seem to limit the width of your lines pretty strictly. There's no real convention in FSS, but can you aim for ~70 on the ones in this PR? Seems weird to have three lines for these and others.